### ISSUE TITLE
Cannot Install on J!3.5 (Related to 66)

### ISSUE THREAD

**Issue by WayLowTech** [JoomGallery/JoomGallery#80]
When I attempt to "Install from Web" or "Upload Package File" J!3.5 returns a blank page with only the following message:

This request has been denied for security reasons. If you believe this was in error, please contact support.

More information:

https://github.com/JoomGallery/JoomGallery/issues/66
More info: http://www.forum.en.joomgallery.net/index.php?topic=7337.0
http://forum.joomla.org/viewtopic.php?f=715&p=3342270

**Comment by Erftralle** [comment]
Can't confirm that.
JoomGallery 3.2.2 installs fine with 'Install from Web' and 'Upload Package File' on a clean Joomla! 3.5.

**Comment by Erftralle** [comment]
@WayLowTech : Still having problems installing JoomGallery on a clean Joomla! ?

**Comment by WayLowTech** [comment]
Actually, I had tried repeatedly to install on this new, clean installation and failed with the same error every time. I searched for a resolution and found the info I posted previously. Soon after I got a reply here I tried again and it worked. I had changed nothing on the Joomla installation other than waiting a couple days and trying again. Since I can no longer reproduce this issue I guess we can safely close it.
Thanks for your follow-up.  


Sent from my Verizon Wireless 4G LTE smartphone

-------- Original message --------
